src/encoded/types/quality_metric.py

```diff
@@ -62,6 +103,10 @@ class QualityMetricFastqc(QualityMetric):
     schema = load_schema('encoded:schemas/quality_metric_fastqc.json')
     embedded_list = QualityMetric.embedded_list
 
+    @calculated_property(schema=QC_SUMMARY_SCHEMA)
+    def quality_metric_summary(self, request):
+        return
```

Comment:
Is this to be implemented?

Reply:
@willronchetti First of all, thanks for the review and your valuable comments. While some QC Item types have quality metric summary, some of them are not having the summary currently. Very first implementation returns an empty list if QC has no summary, but then it is updated to return nothing. Now, removed the redundant quality_metric_summary calc. property. If needed in the future, then will re-add it.